### PR TITLE
CANS-1256 Update readme for Geo Services API project in scope of knowledge transfer

### PR DESCRIPTION
**Pull request by vladimirPe**
CANS-1256 Update readme for Geo Services API project in scope of knowledge transfe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +39,25 @@ The Docker env-file option provides a convenient method to supply these variable
...
 
+## Testing
+
+### Unit Tests
+To run all unit tests, run `./gradlew test`. If the build is successful, all tests passed. If a test fails, you will see more output. If no files have changed, the test run may be very fast.
+
+## Development
+
+### Running the Application
+_Make sure you have configured the valid client id and token for SmartyStreets as well as Perry.
```
Comment (DmitriyRudenko) [inline README.md]: Add link to Perry please.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # CWDS GEO Services API
 
-The CWDS GEO Services API provides RESTful services for the CWDS Digital Services.
+The CWDS GEO Services API provides RESTful API to lookup, suggest, validate and get the distance for physical addresses.
+It uses  SmartyStreets: https://smartystreets.com/ under the hud
```
Comment (DmitriyRudenko) [inline README.md]: Does it work as pure proxy of smartystreets API? If yes, please mention that and provide link to smartystreets API docs.

Comment (vladimirPe) [reply README.md]: Geo services API is a decorator for the SmartyStreets

File: README.md (hunk 1 above)
Comment (vladimirPe) [reply README.md]: link to perry is added

File: README.md (hunk 2 above)
Comment (DmitriyRudenko) [reply README.md]: what kind of decoration does it provide?

**Pull request merged**